server/datastore/mysql/apple_mdm.go

```diff
@@ -3050,7 +3050,7 @@ func subqueryFileVaultVerifying() (string, []interface{}) {
                 AND hmap.profile_identifier = ?
                 AND hmap.operation_type = ?
                 AND (
-		  (hmap.status = ? AND hdek.decryptable IS NULL)
+		  (hmap.status = ? AND hdek.decryptable IS NULL AND hdek.host_id IS NOT NULL)
```

Comment:
Does this cover users that have changed/rotated their filevault keys and they need to restart again? (I'm not sure how we support that scenario)

Reply:
That would be "action required" rather than "verifying", so is covered in the other query. I confirmed that the "log out/restart to rotate" step shows as "action required".